lib/newman_conway.py

```
@@ -1,10 +1,26 @@
 
 
-# Time complexity: ?
-# Space Complexity: ?
+# Time complexity: o(n)
+# Space Complexity: o(n)
 def newman_conway(num):
     """ Returns a list of the Newman Conway numbers for the given value.
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: o(n)
+        Space Complexity: o(n)
     """
-    pass
+    nc_list = [0] * (num + 1)
+    if num == 0:
+        raise ValueError("n must be > 0")
```

We should raise this error for _any_ value below the valid starting point of the sequence:
```py
    if num <= 0:
        raise ValueError("n must be > 0")
```